br/pkg/lightning/mydump/csv_parser.go

```diff
@@ -342,6 +343,7 @@ func (parser *CSVParser) readUntil(chars *byteSet) ([]byte, byte, error) {
 				err = io.EOF
 			}
 			parser.pos += int64(len(buf))
+			log.L().Info("size of buf", zap.Int("size", len(buf)))
```

Comment:
```suggestion
			log.L().Debug("size of buf", zap.Int("size", len(buf)))
```

Not sure in most cases we care about this log

Reply:
Sry. I just forget to cancel this line (it's served for test: https://github.com/pingcap/tidb/issues/40400)

Reply:
5fd6c58ac